sources/packages/web/src/views/aest/AppAEST.vue

```diff
@@ -1,45 +1,60 @@
 <template>
   <IdleTimeChecker :clientIdType="ClientIdType.AEST">
-    <v-app-bar dense flat app style="overflow: visible">
-      <BCLogo
+    <v-app-bar color="white">
+      <b-c-logo
```

Comment:
I'm just curious: there's any reason to change from `<BCLogo>` to `<b-c-logo>`

Reply:
We have a code standard in our project to lean more on the second way.